yocs_safety_controller/src/library.cpp

```diff
@@ -49,7 +49,7 @@ bool SafetyController::init()
 
   enable_controller_subscriber_ = nh_priv_.subscribe("enable", 10, &SafetyController::enableCB, this);
   disable_controller_subscriber_ = nh_priv_.subscribe("disable", 10, &SafetyController::disableCB, this);
-  ranger_subscriber_ = nh_priv_.subscribe("rangers", 10, &SafetyController::rangerCB, this);
+  ranger_subscriber_ = nh_priv_.subscribe("/gopher/bumper", 10, &SafetyController::rangerCB, this);
```

Comment:
Avoid hard coding locations like this as much as possible. Set them private (e.g. "rangers") and let launchers do the remapping.